src/components/Modal/RestaurantDetailModal.jsx

```diff
@@ -0,0 +1,24 @@
+import style from "./Modal.module.css";
```

Comment:
```suggestion
import styles from "./Modal.module.css";
```
`styles`로 변경해 보시는 것은 어떨까요? 다른 모듈들에서 사용해 주실 때 `styles` 이름을 이미 사용하고 계시니 일관성 면에서 통일이 되면 좋을 것 같아요.

그리고, `.module.css` 특성상 여러 개의 스타일들을 주로 담는지라 네이밍도 복수형인 경우가 개발자들 사이에서 훨씬 흔한 것 같아요.